flake8_import_order/__init__.py

```diff
@@ -114,6 +134,11 @@ def _classify_type(self, module):
             elif package in STDLIB_NAMES:
                 return ImportType.STDLIB
 
+        # Check if module's path matches any of given application paths
+        path = get_module_path(module)
```

Comment:
I'm concerned that the underlying functions from importlib/pkgutil will cause needless performance degradation. Would it make sense to nest that in after verifying that we even have any `application_paths` to check?

Reply:
You're right. It seems they try to import parent modules if the target module has sub packages.

I found another solution using `pkgutil.iter_modules`:

```python
names = [name for _, name, _ in 
         pkgutil.iter_modules(self.options.get('application_paths', []))]
```

This runs only once and seems no real import happens. I'll try this.